src/app/tracing/util/labels.ts

```diff
@@ -0,0 +1,40 @@
+
+export const LABELS: Record<string, string> = {
+    id: 'ID',
+    lot: 'Lot',
+    lat: 'Latitude',
+    lon: 'Longitude',
+    dateIn: 'Delivery Date Arrival',
+    dateOut: 'Delivery Date',
+    source: 'Source',
+    target: 'Target',
+    originalSource: 'Original Source',
+    originalTarget: 'Original Target',
+    incoming: 'Incoming',
+    outgoing: 'Outgoing',
+    contains: 'Contains',
+    isMeta: 'Is Meta Station',
+    contained: 'Is Meta Member',
+    forward: 'Forward Trace',
+    backward: 'Backward Trace',
+    traceType: 'Trace Type',
+    observed: 'Observed',
+    outbreak: 'Outbreak',
+    crossContamination: 'Cross Contamination',
+    killContamination: 'Kill Contamination',
+    commonLink: 'Common Link',
+    score: 'Score',
+    weight: 'Weight',
+    anonymisationLabel: 'Anonymisation Label',
+    anonymizedName: 'Anonymized Name',
+    name: 'Name',
+    address: 'Address',
+    country: 'Country',
+    typeOfBusiness: 'Type of Business',
+    selected: 'Selected',
+    invisible: 'Invisible',
+    product: 'Product',
+    amount: 'Amount',
+    stationLabel: 'Station Label',
+    deliveryLabel: 'Delivery Label'
+};
```

Comment:
I know you had declared LABELS to be of type `{ [key: string]: string }`in the beginning and that I replied that you can also use `Record<string, string>`, BUT:

I found out (unfortunately just recently) that:
- `export const LABELS: Record<string, string> = { ... };`  causes VSCode not to list the properties anymore, if you try to use `LABELS`
- `export const LABELS: { [key: string]: string } = { ... };`  causes VSCode to list the properties, if you try to use `LABELS`, but its values in the IDE are only resolved to `string`
- `export const LABELS = { ...  };`  causes VSCode to list the properties, if you try to use `LABELS`, but its values in the IDE are only resolved to `string`
- `export const LABELS = { ...  } as const satisfies Record<string, string>;`  causes VSCode to list the properties, if you try to use `LABELS`. Its values in the IDE are resolved to the specified value

Remark: The keyword `satisfies` was introduced in typescript version 4.9.5

So please use solution 2, 3 or 4 here.

Reply:
My IDE really doesn't like satisfied so I checked the package json and we've got an older version specified: `"typescript": "4.8.4"`. We'd need to bump up the version first. I'll use solution 2 for now and mark it with a comment so we can fix it, once we've updated TypeScript successfully.